source/jormungandr/jormungandr/default_settings.py

```diff
@@ -259,3 +259,4 @@
 BEST_BOARDING_POSITIONS_DIR = os.getenv('JORMUNGANDR_BEST_BOARDING_POSITIONS_DIR', None)
 ORIGIN_DESTINATION_DIR = os.getenv('JORMUNGANDR_ORIGIN_DESTINATION_DIR', None)
 STOP_POINTS_ATTRACTIVITIES_DIR = os.getenv('JORMUNGANDR_STOP_POINTS_ATTRACTIVITIES_DIR', None)
+DEPLOYMENT_AZ_SUFFIX = os.getenv('JORMUNGANDR_DEPLOYMENT_AZ_SUFFIX', "unknown")
```

Comment:
Not sure about the meaning of "suffix" here
```suggestion
DEPLOYMENT_AZ = os.getenv('JORMUNGANDR_DEPLOYMENT_AZ', "unknown")
```

Reply:
Done